### DISCUSSION TITLE
NavigationBar's PrimaryCommands are not working on Android

### DISCUSSION THREAD

**Discussion by WenHong-Lau** [unoplatform/uno#18498] [General]
In this repro, I have added 2 NavigationBar for testing purposes. One in the page itself and another on wrapped in a custom UserControl. I have a click event in the code behind of the Page and UserControl, clicking on these AppBarButtons does not trigger the event. In my own project, I have also tried binding the Command, it does not get triggered as well. This issue happens on **Android** but works fine of WinAppSDK

[UnoApp9.zip](https://github.com/user-attachments/files/17406993/UnoApp9.zip)

**Comment by MartinZikmund** [comment]
Cc @kazo0

**Reply by agneszitte** [reply]
It seems to be related to https://github.com/unoplatform/uno/issues/18506

**Reply by agneszitte** [reply]
@WenHong-Lau I can now confirm that this is the same issue as [uno #18506](https://github.com/unoplatform/uno/issues/18506). 
The problem began with Uno.Sdk 5.4.0-dev.**236** (it is not reproducible with Uno.Sdk 5.4.0-dev.234 or earlier versions). 
We will continue tracking the issue via [uno #18506](https://github.com/unoplatform/uno/issues/18506), and I've provided further details in [this comment](https://github.com/unoplatform/uno/issues/18506#issuecomment-2420139796).

**Reply by agneszitte** [reply]
@WenHong-Lau I am closing this discussion, as it is fixed with latest Uno.Sdk versions.
Details left on the related issue: https://github.com/unoplatform/uno/issues/18506#issuecomment-2631573123
If you notice another issue, don't hesitate to open a new issue, please.
